File: frontend/novalite-todo-portal/src/app/shared/models/TodoAttachment.ts
export interface TodoAttachment{
    $id?:string,
    id?:string,
    todoListId:string,
    type?:string,
    fileName:string
}
export interface TodoAttachmentDTO{
    sasToken:string,
    attachment: TodoAttachment,
    $id?:string,
    type?:string
}

File: backend/NovaLite.Todo.Api/NovaLite.Todo.Core/Models/TodoAttachment.cs
namespace NovaLite.Todo.Core.Models
{
    public class TodoAttachment
    {
        public Guid Id { get; set; }
        public string FileName { get; set; }
        public Guid? TodoListId { get; set; }
        public ToDoList TodoList { get; set; }
        public bool IsDeleted { get; set; }
        public string Type { get; set; }

        public TodoAttachment(string fileName, string type)
        {
            FileName = fileName;
            IsDeleted = false;
            Type = type;
        }
    }
}
